app/assets/stylesheets/user-styles.css

```diff
@@ -33,5 +33,5 @@ img.right {
 }
 
 #sidebar .sidebar-body li {
-  list-style-type: circle;
+  list-style-type: monospace;
```

Comment:
I can't find this in the docs at all :-(

https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type